utils/score_summarization.py

```diff
@@ -0,0 +1,46 @@
+import sys
```

Comment:
In espnet2, we expect to put common scripts in `TEMPLATE/task/pyscripts` or `TEMPLATE/task/script` instead of `utils`. Can you make your changes there?